src/utils/time_utils.hpp

```diff
@@ -19,13 +20,13 @@ namespace time_utils {
 	 * @param min minutes after the hour – [0, 59]
 	 * @param sec seconds after the minute – [0, 61](until C++11) / [0, 60] (since C++11)
 	 */
```

Comment:
comment:

This creates a time without a timezone (e.g. in UTC).
isdst: -1 ...... +1 (add one 1 hour?) 0 .......